packages/apps/patterns/react-ui/src/layouts/PanelPopover/PanelPopover.stories.tsx

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React from 'react';
+
+import '@dxosTheme';
+import { PanelPopover } from './PanelPopover';
+
+export default {
+  component: PanelPopover,
+  actions: { argTypesRegex: '^on.*' }
+};
+
+export const Default = (props: any) => {
+  return <PanelPopover {...props} />;
```

Comment:
Might as well rename these to `Popover` and `Sidebar` given the Dialog renames, right?